src/OSPSuite.SimModel/include/SimModel/Species.h

```diff
@@ -19,14 +20,12 @@ class Species :
 {
 protected:
 	TObjectList<Formula> _rhsFormulaList;
-	//A vector of all formulas that have a reference to this species.
-	std::vector<Formula*> _allFormulaVector;
+	std::vector<EntityWithCachedScaleFactor *> _entitiesWithCachedScaleFactor; //all entities with cached scale factor of THIS species.
 	double m_ODEScaleFactor;
```

Comment:
here all entities which cache the scale factor of the current species are stored